### ISSUE TITLE
CurseForge package wierdness

### ISSUE THREAD

**Issue by b-morgan** [Xruptor/BagSync#394]
Using the CurseForge app, I updated Retail, Cataclysm Classic, and Classic Era to 19.57.

I believe that all three can now use `## Interface: 110005, 110007, 11504, 40401` in the .toc file but only Retail used that format. The other two had a single value in the .toc but it was 40400 and 11502 respectively. The rest of the addon files were identical.

**Comment by b-morgan** [comment]
I think you can remove lines 44-54 in .github/workflows/release.yml and just build one package for all three flavors.

**Comment by Xruptor** [comment]
I'll have to double check that.  In the past it actually didn't separate them and just kept it as Retail.  Which was confusing people.  If it actually creates individual ones that would be great.

If it just puts it as Retail then it has confused people.  Believe it or not I get tickets about, "Why don't you suppose Classic and such..".  When in reality I do but it was filed as Retail with the Interface # update.  So I have to keep it separated for folks whom get confused, since they go directly to download Classic versions and ignore "Retail" even though it's compatible.

**Comment by b-morgan** [comment]
I'm the author / maintainer of Skillet and Skillet-Classic. I just recently changed Skillet-Classic to use one package with
`## Interface: 11504, 40401` in the toc and just one package build in release.yml. The CurseForge app shows Skillet-Classic under both Cataclysm Classic and Classic Era (including SoD).

I believe if you use `## Interface: 11504, 40401, 110005` then the packager and the CurseForge app will do the right thing.

Let me know if I can help.

**Comment by Xruptor** [comment]
Yep I remember you were the author of Skillet 😃.  I'll give it a go next time I do a push and just set the Interface labels and push as a single source.  I believe the other sites WOWInterface.com and Wago.io should be able to handle it.  Worse case scenario I revert it back to the old system.

Thanks again for your suggestion and offer of support, it's much appreciated!

**Comment by Xruptor** [comment]
I have implemented this for BagSync and will apply it to my other addons as well.
